documentation/modules/upgrading/proc-upgrade-brokers-newer-kafka.adoc

```diff
@@ -110,14 +107,32 @@ If required, set the `version` property for Kafka Connect and MirrorMaker as the
 
 . _Optional step to reduce rolling updates_:
```

Comment:
@scholzj - not sure what this step is for? Maybe we can make it clearer?

Reply:
IIRC this was trying to say that when you are upgrading through multiple versions in one go (remember that in the past, to upgrade from Strimzi 0.16 to Strimzi 0.22 you were supposed to go through each of the versions) you do not have to update the `inter.broker.protocol.version` and `log.message.format.version` in every step but only once at the end. I think this is not needed anymore since we now support direct upgrades across multiple versions.

Reply:
I see. I've removed it now